
>>> result = call c (10)
wrong-unbound.hlt:16-17: error, too few arguments given, expected 2 but got 1 [pass::hilti::Validator]
wrong-unbound.hlt: Aborting due to verification error.
error running hiltic, aborting
